2024-08-26 09:05:45,160 INFO    MainThread:216548 [wandb_setup.py:_flush():76] Current SDK version is 0.16.0
2024-08-26 09:05:45,160 INFO    MainThread:216548 [wandb_setup.py:_flush():76] Configure stats pid to 216548
2024-08-26 09:05:45,160 INFO    MainThread:216548 [wandb_setup.py:_flush():76] Loading settings from /home/meganblack/.config/wandb/settings
2024-08-26 09:05:45,160 INFO    MainThread:216548 [wandb_setup.py:_flush():76] Loading settings from /home/meganblack/Project-Build-an-ML-Pipeline-Starter/src/data_check/wandb/settings
2024-08-26 09:05:45,160 INFO    MainThread:216548 [wandb_setup.py:_flush():76] Loading settings from environment variables: {'run_group': 'development', 'project': 'nyc_airbnb'}
2024-08-26 09:05:45,161 INFO    MainThread:216548 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2024-08-26 09:05:45,161 WARNING MainThread:216548 [wandb_setup.py:_flush():76] Could not save program above cwd: /home/meganblack/miniconda3/envs/mlflow-965164e1aea9334f6754c0ab7f0666d1dc3c7cec/bin/pytest
2024-08-26 09:05:45,161 INFO    MainThread:216548 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program_relpath': None, 'program_abspath': '/home/meganblack/miniconda3/envs/mlflow-965164e1aea9334f6754c0ab7f0666d1dc3c7cec/bin/pytest', 'program': '/home/meganblack/miniconda3/envs/mlflow-965164e1aea9334f6754c0ab7f0666d1dc3c7cec/bin/pytest'}
2024-08-26 09:05:45,161 INFO    MainThread:216548 [wandb_init.py:_log_setup():524] Logging user logs to /home/meganblack/Project-Build-an-ML-Pipeline-Starter/src/data_check/wandb/run-20240826_090545-1ozt7bo5/logs/debug.log
2024-08-26 09:05:45,161 INFO    MainThread:216548 [wandb_init.py:_log_setup():525] Logging internal logs to /home/meganblack/Project-Build-an-ML-Pipeline-Starter/src/data_check/wandb/run-20240826_090545-1ozt7bo5/logs/debug-internal.log
2024-08-26 09:05:45,162 INFO    MainThread:216548 [wandb_init.py:init():564] calling init triggers
2024-08-26 09:05:45,162 INFO    MainThread:216548 [wandb_init.py:init():571] wandb.init called with sweep_config: {}
config: {}
2024-08-26 09:05:45,162 INFO    MainThread:216548 [wandb_init.py:init():614] starting backend
2024-08-26 09:05:45,162 INFO    MainThread:216548 [wandb_init.py:init():618] setting up manager
2024-08-26 09:05:45,165 INFO    MainThread:216548 [backend.py:_multiprocessing_setup():105] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2024-08-26 09:05:45,169 INFO    MainThread:216548 [wandb_init.py:init():624] backend started and connected
2024-08-26 09:05:45,174 INFO    MainThread:216548 [wandb_init.py:init():716] updated telemetry
2024-08-26 09:05:45,185 INFO    MainThread:216548 [wandb_init.py:init():749] communicating run to backend with 90.0 second timeout
2024-08-26 09:05:45,608 INFO    MainThread:216548 [wandb_init.py:init():792] run resumed
2024-08-26 09:05:45,646 INFO    MainThread:216548 [wandb_run.py:_on_init():2254] communicating current version
2024-08-26 09:05:45,740 INFO    MainThread:216548 [wandb_run.py:_on_init():2263] got version response upgrade_message: "wandb version 0.17.7 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2024-08-26 09:05:45,741 INFO    MainThread:216548 [wandb_init.py:init():800] starting run threads in backend
2024-08-26 09:05:45,861 INFO    MainThread:216548 [wandb_run.py:_console_start():2233] atexit reg
2024-08-26 09:05:45,862 INFO    MainThread:216548 [wandb_run.py:_redirect():2088] redirect: wrap_raw
2024-08-26 09:05:45,863 INFO    MainThread:216548 [wandb_run.py:_redirect():2153] Wrapping output streams.
2024-08-26 09:05:45,863 INFO    MainThread:216548 [wandb_run.py:_redirect():2178] Redirects installed.
2024-08-26 09:05:45,865 INFO    MainThread:216548 [wandb_init.py:init():841] run started, returning control to user process
2024-08-26 09:05:46,883 INFO    MainThread:216548 [wandb_setup.py:_flush():76] Current SDK version is 0.16.0
2024-08-26 09:05:46,883 INFO    MainThread:216548 [wandb_setup.py:_flush():76] Configure stats pid to 216548
2024-08-26 09:05:46,883 INFO    MainThread:216548 [wandb_setup.py:_flush():76] Loading settings from /home/meganblack/.config/wandb/settings
2024-08-26 09:05:46,883 INFO    MainThread:216548 [wandb_setup.py:_flush():76] Loading settings from /home/meganblack/Project-Build-an-ML-Pipeline-Starter/src/data_check/wandb/settings
2024-08-26 09:05:46,883 INFO    MainThread:216548 [wandb_setup.py:_flush():76] Loading settings from environment variables: {'run_group': 'development', 'project': 'nyc_airbnb'}
2024-08-26 09:05:46,883 INFO    MainThread:216548 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2024-08-26 09:05:46,883 WARNING MainThread:216548 [wandb_setup.py:_flush():76] Could not save program above cwd: /home/meganblack/miniconda3/envs/mlflow-965164e1aea9334f6754c0ab7f0666d1dc3c7cec/bin/pytest
2024-08-26 09:05:46,883 INFO    MainThread:216548 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program_relpath': None, 'program_abspath': '/home/meganblack/miniconda3/envs/mlflow-965164e1aea9334f6754c0ab7f0666d1dc3c7cec/bin/pytest', 'program': '/home/meganblack/miniconda3/envs/mlflow-965164e1aea9334f6754c0ab7f0666d1dc3c7cec/bin/pytest'}
2024-08-26 09:05:46,884 INFO    MainThread:216548 [wandb_init.py:_log_setup():524] Logging user logs to /home/meganblack/Project-Build-an-ML-Pipeline-Starter/src/data_check/wandb/run-20240826_090546-1ozt7bo5/logs/debug.log
2024-08-26 09:05:46,884 INFO    MainThread:216548 [wandb_init.py:_log_setup():525] Logging internal logs to /home/meganblack/Project-Build-an-ML-Pipeline-Starter/src/data_check/wandb/run-20240826_090546-1ozt7bo5/logs/debug-internal.log
2024-08-26 09:05:46,884 INFO    MainThread:216548 [wandb_init.py:init():564] calling init triggers
2024-08-26 09:05:46,884 INFO    MainThread:216548 [wandb_init.py:init():571] wandb.init called with sweep_config: {}
config: {}
2024-08-26 09:05:46,884 INFO    MainThread:216548 [wandb_init.py:init():609] wandb.init() called when a run is still active
2024-08-26 09:05:50,691 WARNING MsgRouterThr:216548 [router.py:message_loop():77] message_loop has been closed
